__tests__/commands/global.js

```diff
@@ -77,3 +82,10 @@ test.concurrent('add without flag', (): Promise<void> => {
     assert.ok(await fs.exists(path.join(config.cwd, 'node_modules', '.bin', 'react-native')));
   });
 });
+
+test.concurrent('add with prefix flag', async (): Promise<void> => {
+  await fs.unlink(tmpGlobalFolder);
+  return runGlobal('add', {prefix: tmpGlobalFolder}, ['react-native-cli'], 'add-with-prefix-flag', async (config) => {
+    assert.ok(await fs.exists(path.join(tmpGlobalFolder, 'bin', 'react-native')));
```

Comment:
See the modification I have made in #1918 to fix the path on different platform
Change this line accordingly:
`-      assert.ok(await fs.exists(path.join(tmpGlobalFolder, 'bin', 'react-native')));`
`+      assert.ok(await fs.exists(path.join(getBinFolder(tmpGlobalFolder), 'react-native')));`